ui/mods/msu/keybinds/key_static.js

```diff
@@ -0,0 +1,138 @@
+MSU.Key = {
+	sortKeyString : function( _key )
+	{
+		var key = '';
+		var keyArray = _key.split('+');
+		var mainKey = keyArray.pop();
+		if (keyArray.length > 1)
+		{
+			keyArray.sort();
+			key = keyArray.reduce(function(a, b) {return a + '+' + b;}) + '+';
+		}
+		else if (keyArray.length == 1)
```

Comment:
does this need a separate else if? I think they might be handled together.

Reply:
Need to test this.

Reply:
Works